solana-programs/anchor/audius-data/tests/follows.ts

```diff
@@ -222,8 +225,64 @@ describe("follows", function () {
       expect(decodedData.followeeHandle.seed).to.deep.equal(
         constants2.handleBytesArray
       );
-      expect(accountPubKeys[0]).to.equal(adminStorageKeypair.publicKey.toString());
-      expect(accountPubKeys[3]).to.equal(newUser1Key.publicKey.toString());
+      expect(accountPubKeys[0]).to.equal(
+        adminStorageKeypair.publicKey.toString()
+      );
+      expect(accountPubKeys[5]).to.equal(newUser1Key.publicKey.toString());
+    });
+
+    it("delegate follows user", async function () {
+      const delegate = await testCreateUserDelegate({
+        adminKeypair,
+        adminStorageKeypair: adminStorageKeypair,
+        program,
+        provider,
+        ...getURSMParams(),
+      });
+
+      // Submit a tx where user 1 follows user 2
+      const followArgs = {
+        accounts: {
+          audiusAdmin: adminStorageKeypair.publicKey,
+          authority: delegate.userAuthorityDelegateKeypair.publicKey,
+          followerUserStorage: delegate.userAccountPDA,
+          followeeUserStorage: userStorageAccount2,
+          userAuthorityDelegate: delegate.userAuthorityDelegatePDA,
+          authorityDelegationStatus: delegate.authorityDelegationStatusPDA,
+        },
+        signers: [delegate.userAuthorityDelegateKeypair],
+      };
+
+      const followTx = await program.rpc.followUser(
+        baseAuthorityAccount,
+        UserActionEnumValues.followUser,
+        { seed: delegate.handleBytesArray, bump: delegate.userBumpSeed },
+        { seed: handleBytesArray2, bump: handle2DerivedInfo.bumpSeed },
+        followArgs
+      );
+
+      const { decodedInstruction, decodedData, accountPubKeys } =
+        await getTransactionWithData(program, provider, followTx, 0);
```

Comment:
can we name `0` as a var to pass in so it's clearer what inputs we're providing to the logic under test?

Reply:
i made this change you requested for pollAccountBalance lmk if that's kind of what you're thinking. personally for me, i'd rather have fewer lines and just hover over the call to see what the definition is if necessary. this would be more widespread so i'd prefer to put this off in a separate PR.